tests/odh/resources/osft_train.py

```
@@ -0,0 +1,67 @@
+    max_seq_len=MAX_SEQ_LEN,
+    max_tokens_per_gpu=MAX_TOKENS_PER_GPU,
+    effective_batch_size=EFFECTIVE_BATCH_SIZE,
+    learning_rate=LEARNING_RATE,
+)
+print("OSFT training completed")
+
+if not os.path.isdir(CKPT_DIR):
+    raise RuntimeError(f"Checkpoint directory not found: {CKPT_DIR}")
+
+ckpt_files = []
+for root, dirs, files in os.walk(CKPT_DIR):
```

Ruff (0.15.21)

[warning] Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)